### ISSUE TITLE
A parameter cannot be found that matches parameter name 'treatTestFailuresAsWarnings'

### ISSUE THREAD

**Issue by AskeHolst** [microsoft/AL-Go#272]
I get an error in my CI/CD:
`RunPipeline action failed. Error: A parameter cannot be found that matches parameter name 'treatTestFailuresAsWarnings'. Stacktrace: at <ScriptBlock>, D:\a\_actions\microsoft\AL-Go-Actions\v2.1\RunPipeline\RunPipeline.ps1: line 343 at <ScriptBlock>, D:\a\_temp\24d261a8-f92c-415e-81bb-6261269bfe39.ps1: line 1 at <ScriptBlock>, <No file>: line 1`

Ia this a new setting that must be added to settings.json?

**Comment by AskeHolst** [comment]
I think I found the problem, AL-Go downloads BcContainerHelper version 4.0.5, but the treatTestFailuresAsWarnings is first added in version 4.0.6.
Maybe add some check on min version required of BcContainerHelper? Just to make issues like this easier to track down. I guess the dependencies on BcContainerHelper are and will be quite high.

More of the output:
```
Downloading BcContainerHelper latest version from CDN
BcContainerHelper version 4.0.5
BcContainerHelper emits usage statistics telemetry to Microsoft
use settings and secrets
Analyzing repository
Checking project dependencies
Checking appDependencyProbingPaths
Warning: No performance test apps found in bcptTestFolders in .AL-Go\settings.json
Warning: No test apps found in testFolders in .AL-Go\settings.json
Project: 
Locating previous release
Invoke Run-AlPipeline
Error: RunPipeline action failed. Error: A parameter cannot be found that matches parameter name 'treatTestFailuresAsWarnings'. Stacktrace: at <ScriptBlock>, D:\a\_actions\microsoft\AL-Go-Actions\v2.1\RunPipeline\RunPipeline.ps1: line 343 at <ScriptBlock>, D:\a\_temp\efbd13fc-ecad-479f-8ddc-5264902092fb.ps1: line 1 at <ScriptBlock>, <No file>: line 1
```

**Comment by freddydk** [comment]
This is a CDN problem - for some reason, it doesn't get 4.0.6.
Will fix this asap.

**Comment by freddydk** [comment]
Should work now - I have patched version 2.1 to grab ContainerHelper from blob storage.
I will probably remove the CDN altogether - this is not the first time I have this problem, sorry
